### ISSUE TITLE
DOCS tabs

### ISSUE THREAD

**Issue by dfushman** [ehb54/nmrsuite#5]
Please add context-dependent (if possible) DOCS tabs to NMRSuite pages.

**Comment by ehb54** [comment]
Context dependent docs now added [here](https://github.com/ehb54/nmrsuite/tree/main/add/docs).

the global docs is `docs/index.html`
for a specific menu group e.g. `analyze` -> `docs/analyze/analyze.html`
for a specific module e.g. `ses` -> `docs/analyze/ses/ses.html`
etc.

Currently populated with `under construction` blanks.
I set these up somewhat by hand, so if one of the links shows "File not found", let me know & i can fix.

n.b. the names are driven by menu:id and module:id, so if they change, the files will need to be renamed.

Once edited in `/opt/genapp/nmrsuite/add/docs`, then `genapp` needs to be run to populate "live" webpages.

Updated both the live nmrsuite & Matthew's dev environment with the templates.
Let me know if you have any questions.
